app/repositories/package.scala

```diff
@@ -42,9 +42,11 @@ package object repositories {
     ReactiveMongoPlugin.mongoConnector.db
   }
 
-  lazy val faststreamPersonalDetailsRepository = new personaldetails.PersonalDetailsMongoRepository()
-  lazy val faststreamContactDetailsRepository = new contactdetails.ContactDetailsMongoRepository()
+  lazy val faststreamPersonalDetailsRepository = new personaldetails.PersonalDetailsMongoRepository
+  lazy val faststreamContactDetailsRepository = new contactdetails.ContactDetailsMongoRepository
+  lazy val schemePreferencesRepository = new schemepreferences.SchemePreferencesMongoRepository
 
+  // Below repositories will be deleted as they are valid only for Fasttrack
   lazy val personalDetailsRepository = new PersonalDetailsMongoRepository()
```

Comment:
It still has dependencies to the old fasttrack project. I'm suggesting to put all fasttrack code to own package in order to avoid class name clashes and to see what else need to be refactored